### ISSUE TITLE
Add project preview GIF

### ISSUE THREAD

**Issue by kaiwalyakoparkar** [kaiwalyakoparkar/spell-bind#37]
Add the gif of the project proview. We can also replace the images of the working project with
- Gif of every level
- Gif of every aspect

**Comment by Deepa-chaurasiya01** [comment]
@kaiwalyakoparkar Sir, may I?

**Comment by kaiwalyakoparkar** [comment]
@Deepa-chaurasiya01 yeah go ahead. Just do one thing then, remove the screenshots and add gifs of each level instead. So there will be 3 gifs in total I believe. If you face any issue let us know :slightly_smiling_face:

**Comment by Deepa-chaurasiya01** [comment]
@kaiwalyakoparkar, sir, Please check my PR.
